Jenkinsfile

```diff
@@ -103,23 +104,23 @@ def init_git() {
 }
 
 def init_git_win() {
-    checkout scm
-    retry(5) {
-        timeout(time: 2, unit: 'MINUTES') {
-            bat 'git submodule update --init -f'
-        }
-    }
+  checkout scm
```

Comment:
Wonder if these format changes are necessary?

Reply:
I think the issue with these is that the Jenkisfile has been formatted properly yet and the automatic linters do not check, so format is inconsistent across the file.